package.json

```diff
@@ -18,12 +19,18 @@
     "@angular/compiler": "11.0.0",
     "@angular/core": "11.0.0",
     "@angular/forms": "11.0.0",
+    "@angular/localize": "11.0.0",
     "@angular/platform-browser": "11.0.0",
     "@angular/platform-browser-dynamic": "11.0.0",
     "@angular/router": "11.0.0",
+    "@ng-bootstrap/ng-bootstrap": "^8.0.0",
+    "bootstrap": "^4.5.0",
```

Comment:
@carryall Should we use the specific latest [bootstrap](https://www.npmjs.com/package/bootstrap) ?
Or because of this 🤔

![Screen Shot 2563-11-18 at 12 25 24](https://user-images.githubusercontent.com/749672/99487530-24db0280-2999-11eb-95b8-ef9f547b3452.png)

Reply:
✅  Fixed in [0510b27](https://github.com/carryall/survey-challenge/pull/2/commits/0510b2712efb07a0111dfe88925c4299876dbd45)